src/main/java/io/thinkit/edc/client/connector/utils/JsonLdUtil.java

```diff
@@ -32,4 +32,8 @@ public static JsonArray deserializeToArray(InputStream body) throws JsonLdError
         var content = jsonDocument.getJsonContent().get();
         return content.asJsonArray();
     }
+
+    public static boolean isSuccessful(Integer value) {
```

Comment:
I'm not sure this is the best place to put this method, I think we have 3 options here:
- define a component that takes care to translate from a `Response` to a `Result` that then can be injected in the controllers
- introduce an abstraction shared between the controllers that will contain the shared logic
- use a new util class related to the http client

we could go with the simplest one for the moment (the third) and create an issue to tackle that (generally `Util` static classes should be avoided as much as possible)